src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -80,28 +79,32 @@ export function HistogramFilter(props: Props) {
 
     if (variable.type === 'number')
       return {
-        binWidth: variable.binWidthOverride ?? variable.binWidth,
+        binWidth: variable.binWidthOverride ?? variable.binWidth ?? 0.1,
         binWidthTimeUnit: undefined,
         independentAxisRange:
           variable.displayRangeMin != null && variable.displayRangeMax != null
             ? { min: variable.displayRangeMin, max: variable.displayRangeMax }
-            : undefined,
+            : { min: Math.min(0, variable.rangeMin!), max: variable.rangeMax! },
```

Comment:
Should we make `{Date,Number}Variable.range{Min,Max}` mandatory props in types/study.ts?

It seems that if `NumberVariable.rangeMax` is missing, the request to the back end will fall over.

If they are missing from the studies/variables metadata we might get a more meaningful error message?

Same goes for date rangeMin and Max - see below